### PR TITLE
Faster integration tests

### PR DESCRIPTION
**Pull request by dgageot**
Fixes https://github.com/docker/machine/issues/2490

### CODE REVIEW

File: test/integration/core/inspect_format.bats [hunk 1]
```diff
@@ -24,7 +23,7 @@ load ${BASE_TEST_DIR}/helpers.bash
 
 @test "$DRIVER: check .Driver output is not flawed" {
   only_if_env DRIVER virtualbox
-  run docker-machine inspect -f '{{.Driver.SSHUser}}' $NAME
+  run machine inspect -f '{{.Driver.SSHUser}}' $NAME
```
Comment (nathanleclaire) [inline test/integration/core/inspect_format.bats]: Nice

**Comment by nathanleclaire** [comment]
Looks great. Why not define a function in `helpers.bash`, e.g. `try_shared_machine` for the little `NAME` check instead of embedding it directly:

```bash
function try_shared_machine {
    if [[ -z "$NAME" ]]; then
        export NAME="$(unique_machine_name)"
    fi
}
```
Then each test would just have one `try_shared_machine` in front.

Would help to reduce duplication.

**Comment by dgageot** [comment]
@nathanleclaire PTAL. I took your feedback into account.

**Comment by nathanleclaire** [comment]
LGTM

**Comment by jeanlaurent** [comment]
LGTM

**Pull request merged**